docs/src/pages/components/text-fields/CustomizedInputBase.tsx

```diff
@@ -11,21 +11,21 @@ export default function CustomizedInputBase() {
   return (
     <Paper
       component="form"
-      sx={{ padding: '2px 4px', display: 'flex', alignItems: 'center', width: 400 }}
+      sx={{ p: '2px 4px', display: 'flex', alignItems: 'center', width: 400 }}
     >
-      <IconButton sx={{ p: '10px' }} aria-label="menu">
+      <IconButton sx={{ p: 1.25 }} aria-label="menu">
         <MenuIcon />
       </IconButton>
       <InputBase
         sx={{ ml: 1, flex: 1 }}
         placeholder="Search Google Maps"
         inputProps={{ 'aria-label': 'search google maps' }}
       />
-      <IconButton type="submit" sx={{ p: '10px' }} aria-label="search">
+      <IconButton type="submit" sx={{ p: 1.25 }} aria-label="search">
         <SearchIcon />
       </IconButton>
       <Divider sx={{ height: 28, m: 0.5 }} orientation="vertical" />
-      <IconButton color="primary" sx={{ p: '10px' }} aria-label="directions">
+      <IconButton color="primary" sx={{ p: 1.25 }} aria-label="directions">
```

Comment:
It seems best to keep hard-coded pixels here. The scaling unit doesn't seem to make sense.

Reply:
@oliviertassinari Would it be for the whole demo or just that line?

Reply:
I think that anywhere that using theme.spacing isn't the intended objective. I believe for the whole demo.

Reply:
Resolved